src/main/resources/ValidationMessages.properties

```diff
@@ -1,3 +1,9 @@
 # this file contains the default messages of JSR 303 bean validation for validations provided in the common library
 
-org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}
\ No newline at end of file
+org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}
+
+org.terasoluna.gfw.common.codepoints.ConsistOf.message = not consists of specified code points
+
+org.terasoluna.gfw.common.validator.constraints.ByteMin.message = must be greater than or equal to {value} bytes
+org.terasoluna.gfw.common.validator.constraints.ByteMax.message = must be less than or equal to {value} bytes
+org.terasoluna.gfw.common.validator.constraints.Compare.message = not match '{left}' and '{right}'
```

Comment:
- [ ] I think "match" does not satisfy the meaning of ``@Compare``. It's just idea, but "invalid combination of {left} and {right}"